packages/@react-stately/calendar/src/useRangeCalendarState.ts

```diff
@@ -177,7 +184,7 @@ export function useRangeCalendarState<T extends DateValue = DateValue>(props: Ra
   };
 }
 
-function makeRange(start: DateValue, end: DateValue): RangeValue<CalendarDate> {
+function makeRange(start: DateValue, end: DateValue): RangeValue<CalendarDate>|null {
   if (!start || !end) {
     return null;
```

Comment:
IMO null should not be used in most cases,  since '?' operator of ts does not recognise it